src/utils/GasUtils.sol

```diff
@@ -48,7 +48,7 @@ library GasUtils {
     /**
      * @dev Base cost of the `IGateway.submitMessage` method.
      */
-    uint256 internal constant SUBMIT_BASE_COST = 24138;
+    uint256 internal constant SUBMIT_BASE_COST = 24138 + 22;
```

Comment:
Maybe need to update the `EXECUTION_SELECTOR_OVERHEAD`, the instructions are in the constant description.